packages/paste-libraries/styling/package.json

```diff
@@ -25,17 +25,15 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "@emotion/core": "10.0.28",
-    "@emotion/styled": "10.0.27",
+    "@emotion/react": "11.9.0",
```

Comment:
`@emotion/react` replaces the following packages (which are now bundled dependencies under this dependency)
- `emotion-theming`
- `@emotion/core`
- `emotion`